src/main/kotlin/no/nav/familie/ef/sak/api/gui/SøkController.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.familie.ef.sak.api.gui
+
+import no.nav.familie.ef.sak.api.dto.PersonIdentDto
+import no.nav.familie.ef.sak.api.dto.Søkeresultat
+import no.nav.familie.ef.sak.service.SøkService
+import no.nav.familie.ef.sak.service.TilgangService
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping(path = ["/api/sok"])
```

Comment:
Borde man kalla denna før personsøk? 
`/api/person/sok/` eller att sokPerson dær nede har `person`

Alternativt ta detta under fagsak, beroende på hur den ska fungera i framtiden

Reply:
ÆR ju en filtrering av fagsak på fnr så jag hade det under fagsakscontroller [her](https://github.com/navikt/familie-ef-sak/pull/507/commits/8aa91d98319e3cabeb0f18d172f98397b13906bd#diff-e55653ba38573d104eb6b847a5f3508538c0a8db1d842dc79899bc37ca62c4b3R32), men så tænkte jag att det var fint att separare det for att det er en egen feature typ och ska inte returnera fagsagDTO 🤔